typings/tests.ts

```diff
@@ -1117,3 +1120,48 @@ client.on('interactionCreate', async interaction => {
 declare const shard: Shard;
 
 assertType<Promise<number | null>>(shard.eval(c => c.readyTimestamp));
+
+// Test audit logs
+declare const guild: Guild;
+assertType<Promise<GuildAuditLogs<'MEMBER_KICK'>>>(guild.fetchAuditLogs({ type: 'MEMBER_KICK' }));
+assertType<Promise<any>>(guild.fetchAuditLogs({ type: 20 }));
```

Comment:
Promise any? That doesn't seem right..

Reply:
Oops it isn't indeed, some leftovers from testings and debugging. Fixed in 2227d1f